src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,23 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += '\n Authorization for identifier %s failed.' % (
+                         authzr.body.identifier)
+                       msg += '\n Here are the challenges that were not fulfilled:'
+                       for challenge in authzr.body.challenges:
+                         msg += \
+                               '\n Challenge Type: %s' \
+                               '\n Error information: ' \
+                               '\n Type: %s' \
```

Comment:
@Ornias1993 can we please use f-strings here ?

Reply:
Sure, i've been out of python for a while though, please check if my newest edit is alright or synthax needs change...